extensions/vai/config.go

```diff
@@ -0,0 +1,9 @@
+package vai
+
+const (
+	ConfigurationFileKey = "vai"
```

Comment:
is this constant going to be used pre dominantly in tests? if so, can  we move this to helper? If its going to b used outside of vai tests we can move this to actions..